source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -321,6 +334,24 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
                 setShowModal(false);
               }}
             />
+
+            {formikRef.current?.values.alternateProject?.id === acquisitionFile.projectId &&
+              showError() && (
+                <GenericModal
+                  display={showAltProjectError}
+                  className="projectError"
+                  title="Alternate Project Error"
+                  message={[
+                    <strong>Error: </strong>,
+                    `You have selected an alternate project that is the same as the file project, please select a different project`,
+                  ]}
```

Comment:
@devinleighsmith I thought we talked about this with dev and it was not necessary anymore. Can you confirm?

Reply:
what we were talking about the other day was the converse - the user changes the primary project to be the same as the alternate project. I think there is still some merit to this case, although having an entire popup for it is probably overkill. However that is what the approved story states so fine for this PR.

Reply:
The controller is returning nulls and I've adjusted the models accordingly but for some reason, the alternate project attribute is still being set as undefined?